core/extensions.rs

```diff
@@ -65,6 +65,12 @@ pub type OpMiddlewareFn = dyn Fn(OpDecl) -> OpDecl;
 pub type OpStateFn = dyn FnOnce(&mut OpState);
 pub type OpEventLoopFn = dyn Fn(Rc<RefCell<OpState>>, &mut Context) -> bool;
 
+/// Trait implemented by all generated ops.
+pub trait Op {
+  const NAME: &'static str;
+  const DECL: OpDecl;
+}
```

Comment:
Great

Reply:
I'll follow-up on this one once the new rusty_v8 lands and make it `Copy`.